HomeWork3/Task2/Program.cs

```diff
@@ -0,0 +1,214 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+internal class Massive
+{
+    public static void Main(string[] args)
+    {
+        string str;
+        bool VariableForTryParse;
+        int timeVariable2 = 0,
+            minValueOfArray,
+            maxValueOfArray,
+            arrayValue,
+            meaning,
+            timeVariable;
+
+
+        while (true)
+        {
+
+
+            Console.WriteLine("\n To exit, type Exit. Otherwise, press any key to create an array");
+            str = Console.ReadLine();
+
+            if (str.ToLower() == "exit")
+            {
+                return;
+            }
+
+            Console.WriteLine("\n Enter the array size no more than 20 and no less than 0:");
+
+
+            VariableForTryParse =
+                int.TryParse(Console.ReadLine(), out arrayValue); //Переменная которая задает размер массива
+
+            if (VariableForTryParse == false || arrayValue > 20 || arrayValue <= 0)
+            {
+                Console.WriteLine("Incorrect value");
+            }
+            else
+            {
+                var array = new int[arrayValue]; //Задаем размер массива
+                Console.WriteLine("Initializing the array:");
+                for (int i = 0; i < array.Length; i++)
+                {
+                    do
+                    {
+                        
+                        VariableForTryParse = int.TryParse(Console.ReadLine(), out meaning);
+                    } while (VariableForTryParse == false);
```

Comment:
It is better to tell the user what went wrong and why you are asking him to enter something again